.github/workflows/unix.yml

```
@@ -218,3 +218,50 @@ jobs:
       - name: Run tests
         run: mkdir -p tmp && python3 -m pytest --verbose --ignore=tests/test_code_conventions.py --ignore=tests/test_kat_all.py ${{ matrix.PYTEST_ARGS }}
         timeout-minutes: 60
+
+  linux_openssl330-dev:
+    needs: buildcheck
+    runs-on: ubuntu-latest
+    container:
+      image: openquantumsafe/ci-ubuntu-jammy:latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Retrieve OpenSSL330 from cache
+        id: cache-openssl330
+        uses: actions/cache@v3
+        with:
+          path: .localopenssl330
+          key: ${{ runner.os }}-openssl330
+      - name: Checkout the OpenSSL v3.3.0 commit
+        if: steps.cache-openssl330.outputs.cache-hit != 'true'
+        uses: actions/checkout@v4
+        with:
+          repository: 'openssl/openssl'
+          ref: '536649082212e7c643ab8d7bab89f620fbcd37f0'
```

It looks like OpenSSL has recently (a few days ago) tagged this commit as `openssl-3.3.0-beta1`. Might as well use that tag here to improve readability.
```suggestion
          ref: 'openssl-3.3.0-beta1'
```